packages/frontend/rollup.config.mjs

```diff
@@ -35,21 +35,23 @@ export default [
                 },
             }),
             peerDepsExternal(),
-            commonjs(),
             nodeResolve({
                 preferBuiltins: false,
                 browser: true,
             }),
+            commonjs(),
+            esbuild({ target: "es2020", tsconfig: "./tsconfig.build.json" }),
             postcss({
                 plugins: [tailwindcss, autoprefixer],
                 extract: resolve("./dist/styles.css"),
             }),
             json(),
-            typescript(),
             image(),
         ],
         output: {
-            dir: resolve("dist"),
+            dir: resolve("./dist"),
+            preserveModules: true,
```

Comment:
Is this actually needed with the new tsconfig?

Reply:
Fixed in 911aae37d4a9927e81cfb9876e0c928171b81f2c.